.github/workflows/linting.yml

```diff
@@ -1,11 +1,10 @@
-name: "Lint"
+name: Lint
 
 on:
   pull_request:
-    branches: [ "*" ]
+    branches: [ '**' ]
```

Comment:
Why the double `*`? Does it do something differently?

Reply:
According to the [GH docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet):

- `*`: Matches zero or more characters, but does not match the `/` character. For example, `Octo*` matches `Octocat`.
- `**`: Matches zero or more of any character.

Reply:
My PRs often have the format `<type>/<description>`, so I don't think it would be due to that as linting works on those PRs fine every time :sweat_smile:

Reply:
Indeed, mine too, but according to the docs it should only have worked with `**`. I will remove the (somewhat useless) restriction entirely as we want the linting to always run on every PR.

Reply:
Done in 9aba16fb20c1dbe5af21c1569faa323062755c71